atomsci/ddm/pipeline/model_wrapper.py

```diff
@@ -2241,7 +2287,8 @@ def generate_predictions(self, dataset):
         pred, std = None, None
         self.log.warning("Evaluating current model")
 
-        pred = self.model.predict(dataset, self.transformers)
+        dataset = self.transform_dataset(dataset, 'final')
+        pred = self.model.predict(dataset, self.transformers['final'])
```

Comment:
It's a trade off between compute time and space in RAM. Either cache it or compute it over and over again. This would be simpler if we move k-fold outside of the pipeline.